src/main/scala/com/datastax/gatling/plugin/utils/TimingUtils.scala

```diff
@@ -0,0 +1,53 @@
+package com.datastax.gatling.plugin.utils
+
+import java.util.concurrent.TimeUnit.{MILLISECONDS, NANOSECONDS}
+
+import io.gatling.commons.util.ClockSingleton
+import io.gatling.core.Predef.Session
+
+import scala.concurrent.duration.TimeUnit
+
+/**
+  * This class is a hack that allows to measure latency in nanoseconds.
+  *
+  */
+// TODO: give back the improvements in this class to Gatling by submitting a PR to add `DefaultClock::computeTimeFromNanos(nanos: Long, targetUnit: TimeUnit)`
```

Comment:
Does this belong in the code?

Reply:
My rationale was to emphasize that it is a bad hack and we should eventually eliminate it.  When I wrote it, I had in mind what the gatling changes needed to be so I wrote them here in order not to forget.